deploy/012-deploy-prime.ts

```diff
@@ -32,6 +32,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     arbitrumone: NINETY_DAYS,
     zksyncmainnet: NINETY_DAYS,
     opmainnet: NINETY_DAYS,
+    basesepolia: NINETY_DAYS,
```

Comment:
We are usually set ten minutes for Prime on testnets. Moreover, I would put this entry after the `opsepolia` entry, with the rest of the testnets entries

Reply:
I think we'll need to redeploy the Prime contract, because this data is stored in an immutable variable